src/bindgen/ir/enumeration.rs

```diff
@@ -208,9 +209,15 @@ impl EnumVariant {
 impl Source for EnumVariant {
     fn write<F: Write>(&self, config: &Config, out: &mut SourceWriter<F>) {
         let condition = self.cfg.to_condition(config);
+        let escaped_export_name = {
+            let mut name = self.export_name.to_owned();
+            reserved::escape(&mut name);
```

Comment:
`write()` is not the right place to do this. This should be done in `rename_for_config` for consistency with the other callers.

Reply:
Oh I see, never know about the `Itme` trait. I'll change it!